its/src/test/kotlin/org/sonarlint/intellij/its/tests/FocusOnNewCodeTest.kt

```diff
@@ -0,0 +1,198 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2023 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.its.tests
+
+import com.intellij.remoterobot.RemoteRobot
+import com.sonar.orchestrator.Orchestrator
+import com.sonar.orchestrator.container.Edition
+import com.sonar.orchestrator.locator.FileLocation
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.condition.DisabledIf
+import org.sonarlint.intellij.its.BaseUiTest
+import org.sonarlint.intellij.its.fixtures.idea
+import org.sonarlint.intellij.its.fixtures.tool.window.toolWindow
+import org.sonarlint.intellij.its.utils.OrchestratorUtils
+import org.sonarlint.intellij.its.utils.ProjectBindingUtils
+
+@DisabledIf("isCLionOrGoLand", disabledReason = "No Java Issues in CLion or GoLand")
+class FocusOnNewCodeTest : BaseUiTest() {
+
+    @Test
+    fun should_display_sink() = uiTest {
+        openExistingProject("sample-java-taint-vulnerability", true)
+        bindProjectFromPanel()
+
+        openFile("src/main/java/foo/FileWithSink.java", "FileWithSink.java")
+
+        verifyReportTabContainsMessages(
+            this,
+            "Found 2 new issues in 1 file  (unsupported new code definition)",
+            "No older issues",
+            "Found 1 new Security Hotspot in 1 file  (unsupported new code definition)",
+            "No older Security Hotspots"
+        )
+
+        verifyTaintTabContainsMessages(
+            this,
+            "Found 1 new issue in 1 file  (unsupported new code definition)",
+            "FileWithSink.java",
+            "Change this code to not construct SQL queries directly from user-controlled data.",
+            "No older issues"
+        )
+
+        verifyHotspotTabContainsMessages(
+            this,
+            "Found 1 new Security Hotspot in 1 file  (unsupported new code definition)",
+            "No older Security Hotspots"
+        )
+
+        verifyIssueTreeContainsMessages(
+            this,
+            "Found 2 new issues in 1 file  (unsupported new code definition)",
```

Comment:
Does it make sense to test for `(unsupported new code definition)`? It kinda means there is an issue no? 😅